files/__addonLocation__/rollup.config.mjs

```diff
@@ -15,7 +15,11 @@ export default {
   plugins: [
     // These are the modules that users should be able to import from your
     // addon. Anything not listed here may get optimized away.
-    addon.publicEntrypoints(['components/**/*.js', 'index.js'<% if (typescript) {%>, 'template-registry.js'<% } %>]),
+    addon.publicEntrypoints([
```

Comment:
important change -- no more typescript branches in rollup. there is no downside to having this stuff here

Reply:
Not sure I agree with this change... 

First, seems unrelated to the PR's primary purpose?

Then, I would find it confusing to see a file referenced that does not exist, and that does not even have any use (in a JS addon). Would be different if it were reference implicitly through some glob like `*.js`, but here we have an explicit reference to this specific file.

I'd rather have a bit of complexity on our side, than cause confusion to the user...

Reply:
it's related in that we emit a prettier error in TS projects, but I think I can just wrap the whole line rather than have the typescript condition inline